data/json/bionics.json

```diff
@@ -877,7 +873,7 @@
     "type": "bionic",
     "name": { "str": "Fingertip Razors" },
     "description": "You possess razor-sharp claws underneath your fingernails that do a small amount of unarmed slashing damage whenever your fingertips are uncovered.",
-    "occupied_bodyparts": [ [ "hand_l", 2 ], [ "hand_r", 2 ] ],
+    "occupied_bodyparts": [ [ "hand_l", 1 ], [ "hand_r", 1 ] ],
     "fake_item": "fake_razor",
```

Comment:
This should either be BIONIC_TOGGLED (with full slot usage) or give encumbrance (since you'd be needing to be careful not to scratch things you didn't want to).

Reply:
There are bionic claws, that are exactly what you're suggesting.
These are some pressure-activated/permanent short blades under fingernails. Sort of long nails but cyberpunk.